docs/src/main/asciidoc/neo4j.adoc

```diff
@@ -4,10 +4,9 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 ////
 = Quarkus - Neo4j
-:neo4j_version: 3.5.13
+:neo4j_version: 4.0.0
 
 include::./attributes.adoc[]
-:extension-status: preview
```

Comment:
I would keep the extension as `preview` for now until we get sufficient feedback.

Agreed to remove the warning below.

Reply:
Agreed and added back in.